compiler/crates/relay-transforms/src/apply_transforms.rs

```diff
@@ -122,14 +122,25 @@ fn apply_common_transforms(
     let log_event = perf_logger.create_event("apply_common_transforms");
     log_event.string("project", project_config.name.to_string());
     let mut program = log_event.time("transform_connections", || {
-        transform_connections(&program, &project_config.schema_config.connection_interface)
+        transform_connections(
+            &program,
+            &project_config.schema_config.connection_interface,
```

Comment:
I think we should consider adding `schema_config` directly to the `program`.

We already have `Arc<Schema>` there. And I think it would be easier if we can just always have them together: `schema` and `schema_config`.

Let me try this internally.

Reply:
Update: I could not come up with a better version of the Schema with Config. We have some some non-relay users of Programs/Schema, so adding a dependency on `relay-config` doesn't make sense for them. And adding something like a trait for `Program` is a bigger change. 

Let's keep it for now, but maybe in this specific line: let's pass a `&project_config.schema_config` to the `transform_connections` instead of individual config options.

Reply:
@alunyov I've updated `transform_connections` to accept schema_config